Companies add topic

HenryPajuri opened the pull request.

rkolktlu committed:
Companies-addTopic Branch Init

Frontend & backend alternations. Added button to add topics, created routes for the topic registration page. Created a form for submitting topics and created urls and triggers to register a topic.

rkolktlu committed:
alter

removed un-necessary files

HenryPajuri committed:
minor fixes

romilrobtsenkov reviewed (commented):
Currently code has many unneeded changes and does not run. Did not review. Please fix ASAP.

HenryPajuri committed:
Fixed some files

I managed to get the main site running again, but i stlll had some problems with the URL of topics/add..

romilrobtsenkov commented:
Does not fulfill requirements and changes were not made on time
